pkg/network/common/common.go

```diff
@@ -35,14 +35,15 @@ func ClusterNetworkListContains(clusterNetworks []ClusterNetwork, ipaddr net.IP)
 type NetworkInfo struct {
 	ClusterNetworks []ClusterNetwork
 	ServiceNetwork  *net.IPNet
+	VXLANPort       *uint32
```

Comment:
We definitely should make it non-optional and defaulted by this point. Maybe have `ParseNetworkInfo` take a pointer since that's convenient for its callers, but then have it substitute in the default value if it's `nil` and then `NetworkInfo` will have a non-pointer field